app/react/Layout/SidePanel.js

```diff
@@ -16,6 +20,11 @@ SidePanel.propTypes = {
   children: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
   className: PropTypes.string,
   open: PropTypes.bool,
+  fixed: PropTypes.bool,
+};
+
+SidePanel.defaultProps = {
+  fixed: false,
```

Comment:
I'm not sure I understand the semantics of this prop?  Is this only passed when the sidepanel is not allowed to be closed?  What does fixed mean?

Because on the entity view, we also have a 'fixed' side panel, yet we are probably not going to flag that with table-vew-mode.  Should this have a different name?

Reply:
That's the semantic, yes. Again, we tried to balance :D We can agree there's a mix between abstraction and ad-hoc design here. We can go with a deeper refactor if you think it makes sense.

Reply:
Ok.